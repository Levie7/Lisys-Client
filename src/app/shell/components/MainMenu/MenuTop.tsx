import * as React from 'react';

import { Picture } from 'src/shared/components/Picture';

import { logo } from './constants';
import { Menu } from './Menu';

export class MenuTop extends React.Component<{}> {
    render() {
        return (
            <div className='d-flex'>
                <Picture
                    alt={logo.light.alt}
                    className='bg-primary px-4'
                    height='46px'
                    src={logo.light.url}
                    title={logo.light.title}
                />
                <Menu className='w-100' mode='horizontal' theme='dark' />
            </div>
        );
    }
}
