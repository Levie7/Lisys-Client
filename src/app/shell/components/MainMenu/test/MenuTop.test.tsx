import { shallow } from 'enzyme';
import React from 'react';

import { MenuTop } from '../MenuTop';

describe('MenuTop', () => {
    const wrap = shallow(<MenuTop />);

    it('<MenuTop/> - should render MenuTop component correctly', () => {
        expect(wrap).toMatchSnapshot();
    });

    it('should render drawer wrapper component', () => {
        expect(wrap.find('#MenuTop').exists()).toBeTruthy();
    });
});
