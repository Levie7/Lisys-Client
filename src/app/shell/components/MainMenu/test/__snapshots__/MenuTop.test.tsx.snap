// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuTop <MenuTop/> - should render MenuTop component correctly 1`] = `ShallowWrapper {}`;
